websites/T/TruckersMP/dist/metadata.json

```diff
@@ -1,30 +1,30 @@
 {
   "author": {
-    "name": "Bas950",
-    "id": "241278257335500811"
```

Comment:
Add as contributor?

Reply:
Nah, it's a presences Bas is assigned too, but hasn't made a contribution towards it. The original dev left a while ago I assume